config/styleguide.config.json

```diff
@@ -39,6 +39,7 @@
     "name": "Components",
     "description": "React components",
     "components": [
+      "StyledArticleTagsSelector",
```

Comment:
I think this is a prime opportunity to do some reorganization.
1) Create a new section for styled-components. Practically most apps will use one or the other, not both, so they probably shouldn't be mixed
2) Move this into a new section as it really isn't a component but a full feature. Not sure of a good name though. Maybe "view"?

Reply:
Yeah, I have a couple of ideas - maybe even  atoms > molecules > organisms (http://bradfrost.com/blog/post/atomic-web-design/)